credentials/alts/alts.go

```diff
@@ -137,19 +137,24 @@ type altsTC struct {
 }
 
 // NewClientCreds constructs a client-side ALTS TransportCredentials object.
-func NewClientCreds(opts *ClientOptions) credentials.TransportCredentials {
+func NewClientCreds(opts *ClientOptions) (credentials.TransportCredentials, error) {
 	return newALTS(core.ClientSide, opts.TargetServiceAccounts, opts.HandshakerServiceAddress)
 }
 
 // NewServerCreds constructs a server-side ALTS TransportCredentials object.
-func NewServerCreds(opts *ServerOptions) credentials.TransportCredentials {
+func NewServerCreds(opts *ServerOptions) (credentials.TransportCredentials, error) {
 	return newALTS(core.ServerSide, nil, opts.HandshakerServiceAddress)
 }
 
-func newALTS(side core.Side, accounts []string, hsAddress string) credentials.TransportCredentials {
+func newALTS(side core.Side, accounts []string, hsAddress string) (credentials.TransportCredentials, error) {
+	var err error
+	err = nil
```

Comment:
`nil` is the zero value for `error`. Why is this line required?

Reply:
Yeah I think it's not necessary. Fixed in a new PR: https://github.com/grpc/grpc-go/pull/3904. We can continue our discussion there.